README.md

```diff
@@ -1,4 +1,4 @@
-[![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)
+![beta](https://img.shields.io/badge/stability-experimental-red.svg)
```

Comment:
why beta instead of experimental? The badge says experimental

Reply:
I made a mistake.